helm/trivy/values.yaml

```diff
@@ -4,7 +4,7 @@ fullnameOverride: ""
 image:
   registry: docker.io
   repository: aquasec/trivy
-  tag: 0.13.0
+  tag: 0.14.0
```

Comment:
@czunker  Remember to bump up `appVersion` from `0.13.0` to `0.14.0` and `version` from `0.1.0` to `0.2.0` in `Chart.yaml`

Actually we could have a template function that reads the tag from the `appVersion` property, WDYT?

Reply:
Bumped the versions.

Reply:
I'm not sure about the function.